<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1489</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qkinz</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/resorces.qrc">
    <normaloff>:/media/Qkinz.icns</normaloff>:/media/Qkinz.icns</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="settingWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="Layout_beamInfo">
             <item>
              <widget class="QLabel" name="BeamTypeLabel">
               <property name="text">
                <string>Current beam:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentBeam">
               <property name="text">
                <string>A^B</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutBeamEnergy">
             <item>
              <widget class="QLabel" name="EnergyLabel">
               <property name="text">
                <string>Beam energy(MeV):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentEnergy">
               <property name="text">
                <string>20</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ChangeBeamButton">
             <property name="text">
              <string>Change beam</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutTarget">
             <item>
              <widget class="QLabel" name="TargetLabel">
               <property name="text">
                <string>Target:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentTarget">
               <property name="text">
                <string>A^B</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutThickness">
             <item>
              <widget class="QLabel" name="ThicknessLabel">
               <property name="text">
                <string>Target thickness:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentThickness">
               <property name="text">
                <string>2.50</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TargetUnitLabel">
               <property name="text">
                <string>mg/cm2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ChangeTargetButton">
             <property name="text">
              <string>Change target</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="FrontLabel">
               <property name="text">
                <string>Target front:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentFront">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="FrontWidthLabel">
               <property name="text">
                <string>Front width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="FrontWidth">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="FrontUnitLabel">
               <property name="text">
                <string>mg/cm2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="BackLabel">
               <property name="text">
                <string>Target back:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentBack">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="BackWidthLabel">
               <property name="text">
                <string>Back width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="BackWidth">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="BackUnitLabel">
               <property name="text">
                <string>mg/cm2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ChangeFBButton">
             <property name="text">
              <string>Change front/back</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layoutDEmat">
             <item>
              <widget class="QLabel" name="dEMatLabel">
               <property name="text">
                <string>dE detector material:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dEMatSym">
               <property name="text">
                <string>Si</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="dEdetLayout">
             <item>
              <widget class="QLabel" name="dEdetLabel">
               <property name="text">
                <string>dE detector width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentDE">
               <property name="text">
                <string>150</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dEdetUnitLabel">
               <property name="text">
                <string>um</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layoutEmat">
             <item>
              <widget class="QLabel" name="EMatLabel">
               <property name="text">
                <string>E detector material:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="EMatSym">
               <property name="text">
                <string>Si</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutEdet">
             <item>
              <widget class="QLabel" name="EdetLabel">
               <property name="text">
                <string>E detector width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentE">
               <property name="text">
                <string>1550</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="EdetUnitLabel">
               <property name="text">
                <string>um</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutAbsMat">
             <item>
              <widget class="QLabel" name="AbsMatLabel">
               <property name="text">
                <string>Absorber material:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="AMatSym">
               <property name="text">
                <string>Al</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutAbs">
             <item>
              <widget class="QLabel" name="AbsWLabel">
               <property name="text">
                <string>Absorber width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CurrentA">
               <property name="text">
                <string>10.5</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="AbsUnitLabel">
               <property name="text">
                <string>um</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ChangeTelescopeButton">
             <property name="text">
              <string>Change telescope</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LayoutStrip">
             <item>
              <widget class="QLabel" name="StripLabel">
               <property name="text">
                <string>Strip:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="StripNumbr">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>7</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Fragment:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="protons">
                 <property name="text">
                  <string>Protons</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="deutrons">
                 <property name="text">
                  <string>Deutrons</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="tritons">
                 <property name="text">
                  <string>Tritons</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="He3s">
                 <property name="text">
                  <string>He-3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="alphas">
                 <property name="text">
                  <string>α's</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="AngleLabel">
                 <property name="text">
                  <string>Angle:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="FwdAngRButton">
                 <property name="text">
                  <string>47°</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="BwdAngRButton">
                 <property name="text">
                  <string>133°</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="RunButton">
                 <property name="text">
                  <string>Run</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ResetButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="OCLlogo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../resources/resorces.qrc">:/media/clab-logo-med.gif</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QCustomPlot" name="plotTab">
         <attribute name="title">
          <string>Plot</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tableTab">
         <attribute name="title">
          <string>Table</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QWebView" name="webView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1489</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExport_plot"/>
    <addaction name="actionExport_table"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout_QCustomPlot"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_plot">
   <property name="text">
    <string>Export plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionExport_table">
   <property name="text">
    <string>Export table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout_QCustomPlot">
   <property name="text">
    <string>QCustomPlot</string>
   </property>
   <property name="toolTip">
    <string>stuff</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/resorces.qrc"/>
 </resources>
 <connections/>
</ui>
